client.go

```diff
@@ -159,6 +158,10 @@ func (c *Client) sendScalingMsg(data *logMetrics) {
 		"prefix": *data.prefix,
 	}).Debug("sendScalingMsg")
 
+	for _, v := range data.events {
+		c.SimpleEvent("Heroku app: "+*data.app, v)
```

Comment:
Fixed and this can be ok for first iteration, we can improve after we have some data.